.github/workflows/ci.yml

```diff
@@ -258,7 +258,8 @@ jobs:
           npm run clean
           npm run build
           # Note: this modifies package.json to test puppeteer-core, so we test this last.
-          npm run test:install
+          # TODO(jrandolf): Disabled for now.
```

Comment:
Is it solved in the same way as the docker test issue? I'd suggest fixing the docker and bundling/installation tests as part of this PR.

Reply:
The docker and install tests are similar. I think it would be better to keep the PR as is to scope it to the separation.

Reply:
I think it's better to author it as a second commit in this PR but it's better to keep the test coverage